src/lib/input/input-container.html

```diff
@@ -30,6 +36,6 @@
           [class.md-warn]="dividerColor == 'warn'"></span>
   </div>
 
-  <div *ngIf="hintLabel != ''" [attr.id]="_hintLabelId" class="md-hint">{{hintLabel}}</div>
+  <div *ngIf="hintLabel != ''" [attr.id]="_hintLabelId" class="md-hint, mat-hint">{{hintLabel}}</div>
```

Comment:
Shouldn't `md-hint` be replaced with `mat-hint` rather than appended? Either way, comma should be removed.